data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -162,6 +162,9 @@ then
 		--volume "$PWD:/assets:z" \
 		"${CONFIG_OPERATOR_IMAGE}" \
 		/usr/bin/cluster-config-operator render \
+		--cloud-provider-config-input-file=/assets/manifests/cloud-provider-config.yaml \
+		--cluster-infrastructure-input-file=/assets/manifests/cluster-infrastructure-02-config.yml \
+		--cloud-provider-config-output-file=/assets/config-bootstrap/cloud-provider-config-generated.yaml \
```

Comment:
can we update the Machine Config Operator's render to point to the new generated file here https://github.com/openshift/installer/blob/master/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L296

This would test the goal of bootstrap generation.